src/simple-filepicker.android.ts

```diff
@@ -65,3 +58,14 @@ export const openFilePicker = (params?: FilePickerOptions) => {
         }
     });
 };
+
+const getTypes = (extensions: string[] = []) => {
+    let types = '*/*';
+    for (let i = 0; i < extensions.length; i++) {
+        types += extensions[i];
```

Comment:
it looks to me you are concatenating over '*/*'. We should have the types variable initialized to an empty string before constructing the extensions string.

Reply:
Good spot, I've amended to only return '*/*' when no extensions are provided.